src/test/java/MyTest.java

```diff
@@ -0,0 +1,22 @@
+import com.files.ValidateImages;
+import com.image.AllImages;
+import com.image.Images;
+import com.image.UrlImage;
+import com.image.User;
+import org.junit.Test;
+
+import java.net.MalformedURLException;
+
+public class MyTest {
+
+    @Test
+    public void test1() throws MalformedURLException {
```

Comment:
@PavelNejinskiy the test should also check if added image can be retrieved back

Reply:
@PavelNejinskiy please follow this blog post on test methods naming
http://www.yegor256.com/2014/04/27/typical-mistakes-in-java-code.html#test-method-names